event_push/event_push.py

```diff
@@ -544,34 +712,76 @@ def worker_process(args):
         return
     print_summary_info(logger, edge_vars, main_vars, if_config_vars)
 
-    if not c_config['testing']:
-        # check project name first
-        check_success = check_project_exist(logger, main_vars)
-        if not check_success:
-            return
+    # TODO: get debug info
+    get_debug_info(logger, c_config, edge_vars, main_vars, if_config_vars)
+
+    if c_config['debug-project']:
```

Comment:
Need to add the list-projects condition here. Currently the list-projects will continue to process the data after the debug call is made.

Reply:
> Need to add the list-projects condition here. Currently the list-projects will continue to process the data after the debug call is made.

Added